<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuddySessionWidget</class>
 <widget class="QWidget" name="BuddySessionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buddyAvatar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buddyInfo">
       <item>
        <widget class="QLabel" name="buddyName">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buddyDetails">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="chatPage">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTextEdit" name="chattedText">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="chatInput">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="callPage">
      <layout class="QVBoxLayout" name="verticalLayout_7"/>
     </widget>
     <widget class="QWidget" name="videoPage">
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </widget>
     <widget class="QWidget" name="screenPage">
      <layout class="QVBoxLayout" name="verticalLayout_5"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="communicationButtons">
     <item>
      <widget class="QPushButton" name="chatButton">
       <property name="text">
        <string>Chat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="callButton">
       <property name="text">
        <string>Call</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="videoCallButton">
       <property name="text">
        <string>Video</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="screenShareButton">
       <property name="text">
        <string>Screen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
